assets/css/detours/_detour_map.scss

```diff
@@ -1,5 +1,15 @@
+.c-detour_map--original-route-shape-skeleton {
```

Comment:
💀 I'd love to come up with a better name for this.

My original idea was to call this `original-route-shape` and the other one `original-route-shape-outline`, but the other one is really standing in for the _actual_ route shape, since that's where all the actual behavior is. This one is just a visual marker showing where the route shape is when it's not highlighted.

Reply:
Just a random thought, how about "core" as opposed to "skeleton?" Not sure if that's any better.

Reply:
I think it's better than "skeleton" at least!